cmd/fyne_demo/screens/widget.go

```diff
@@ -30,11 +30,21 @@ func makeButtonTab() fyne.Widget {
 	disabled := widget.NewButton("Disabled", func() {})
 	disabled.Disable()
 
-	grid := widget.NewTextGridFromString("TextGrid\n  Content")
+	grid := widget.NewTextGridFromString("TextGrid\n  Content\nZebra")
 	grid.SetStyleRange(0, 0, 0, 3,
 		&widget.CustomTextGridStyle{FGColor: color.RGBA{R: 0, G: 0, B: 128, A: 255}})
 	grid.SetStyleRange(0, 4, 0, 7,
 		&widget.CustomTextGridStyle{BGColor: &color.RGBA{R: 128, G: 0, B: 0, A: 255}})
+	grid.Content[1].Style = &widget.CustomTextGridStyle{BGColor: &color.RGBA{R: 64, G: 64, B: 64, A: 128}}
+
+	white := &widget.CustomTextGridStyle{FGColor: &color.RGBA{R: 255, G: 255, B: 255, A: 255}}
+	black := &widget.CustomTextGridStyle{FGColor: &color.RGBA{R: 0, G: 0, B: 0, A: 255}}
+	grid.Content[2].Content[0].Style = white
```

Comment:
Content Content is a bit confusing
```suggestion
	grid.Row[2].Cell[0].Style = white
```

Reply:
Yes, I could not decide if consistency worked here.
Do you have a good suggestion? I originally had "Cells" but it seemed a little uninspired...

Reply:
Oh, haha I see your suggestion now sorry. It’s cunning. But if the index were row and col then perhaps plurals read better? Grid.Rows[row].Cells[col]?